client/src/components/TransferForm.vue

```diff
@@ -76,20 +98,27 @@ export default {
     createSetCurrentValueFunc(transferType, varName) {
       return (newValue) => {
         // Set the new value temporarily
-        this.transferParams[transferType][varName] = newValue;
+        this.allTransferData[transferType][varName] = newValue;
+        this.transferData[varName] = newValue;
 
         // Set the value for good.
-        this.setTransferParams(this.transferParams);
+        this.$emit('updateTransfer', this.transferData);
       };
     },
+    doSomething() {
+      console.log('rendered');
+    },
```

Comment:
Probably not intended for this to be here?

Reply:
Good find